spec/models/application_record_spec.rb

```diff
@@ -67,6 +67,21 @@
     end
   end
 
+  describe 'data_rows' do
+    let :call do
+      @record.data_rows
+    end
+    before :each do
+      @record = create :application_record,
+                       data: [['a question', 'an answer']]
```

Comment:
>Not at the time it was written

Not at the time *what* was written? It looks like `data_rows` is expecting a 4-element array to be each element of data types. You just wrote these tests.

We can remove import tasks and related methods.